File: backend/components/metiers/threed/fixtures/core/samples/users/sample_users.js
/* global console */
// sample_users.js – Exemple ultra avancé de génération d'utilisateurs (fixtures/core)
const { generateUser } = require('../fixtures.generator');
const { auditFixture } = require('../audit');
const { rgpdSanitize } = require('../rgpd');

function sampleUsersUltra() {
  // Génération d'un utilisateur avec audit, RGPD, accessibilité
  let user = generateUser('superadmin');
  user = rgpdSanitize(user);
  auditFixture(user, 'generate');
  console.info('Utilisateur généré:', user);
  console.log('Utilisateur ultra avancé généré avec succès.');
}

module.exports = sampleUsersUltra;

File: backend/components/metiers/threed/fixtures/core/samples/users/sample_users.py
# sample_users.py – Exemple ultra avancé de génération d'utilisateurs (fixtures/core)
from ....generators.fixtures_generator import generate_user
from ....helpers.helpers import audit_fixture, anonymize_fixture
import logging

def rgpd_sanitize(user):
    # Stub RGPD : anonymise l'utilisateur pour la conformité RGPD
    return anonymize_fixture(user)

def sample_users_ultra():
    # Génération d'un utilisateur avec audit, RGPD, accessibilité
    user = generate_user()
    user = rgpd_sanitize(user)
    audit_fixture(user)
    logging.info(f"Utilisateur généré: {user}")
    print('Utilisateur ultra avancé généré avec succès.')
